src/main/java/ca/mcgill/ecse428/parkingsystem/controller/ReservationController.java

```diff
@@ -42,14 +42,18 @@ public Reservation addReservation(@RequestBody Reservation rsv) {
 		return repository.addReservation(rsv);
 	}
 	
-	// Need to check if annotations are correct!! 
-	@PostMapping(consumes = "application/json", produces = "application/json")
-	public Boolean deleteReservationFromTable(@PathVariable String pKey) {
-		return repository.deleteReservationFromTable(pKey);
+	@DeleteMapping(path = "/delete/{pKey}")
+	public ResponseEntity<String> deleteReservation(@PathVariable String pKey) {
+		boolean isDeleted = repository.deleteReservation(pKey);
+		if(isDeleted) {
+			return new ResponseEntity<String>("Reservation successfully deleted.", HttpStatus.OK);
```

Comment:
Maybe change the return status to 204 No Content.

Reply:
Yeah, I think you have a good point, we can remove cancel.